app/javascript/sidebar-widget/SidebarWidget.jsx

```diff
@@ -27,22 +27,24 @@ class SidebarWidget extends Component {
     });
   }
 
-  getSuggestedUsers() {
+  async getSuggestedUsers() {
```

Comment:
I saw babel-polyfill inside the yarn file. So I used async/await. Should I change it to then/catch?